posting/lists.go

```diff
@@ -195,6 +195,59 @@ func (lc *LocalCache) getInternal(key []byte, readFromDisk bool) (*List, error)
 	return lc.SetIfAbsent(skey, pl), nil
 }
 
+func (lc *LocalCache) GetSinglePosting(key []byte) (*pb.PostingList, error) {
```

Comment:
since its a public function could you add a description of what this function's doing/why its necessary? basically what you explained yesterday in text format would be helpful for future reference